package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appoptics-bindings",
-  "version": "10.0.0",
+  "version": "10.0.0-lambda-4",
```

Comment:
In semantic versioning this would be considered lower precedence:
> When major, minor, and patch are equal, a pre-release version has lower precedence than a normal version: Example: 1.0.0-alpha < 1.0.0.

Reply:
yes, it won't go out the door with the suffix. i use that during development to make sure that the correct versions of each component are running; i increment the suffix number each time i incorporate new code into a layer so that there is a record of what is in a given layer.

when i get around to actually releasing  the `-lambda-x` suffix will go away and this will version 10.0.0.

Reply:
actually, i've changed the strategy a bit. i've added an appoptics.version-suffix property to package.json and when building a layer the code replaces version with version + appoptics.version-suffix. so lambda releases will always have a suffix but it won't be part of the version that is released to a generic node environment.